### PR TITLE
gen3: bood domd from sd

### PR DESCRIPTION
**Pull request by Deedone**
Disable eMMC passthrough. Update IPL and DTB with proper boot parameters.

### CODE REVIEW

**Comment by xakep-amatop** [comment]
(note) it is possible that Zephyr Dom0 is also will use SD cards slots, i.e:
* mmc@ee100000 --- h3ulcb & salvator xs
* mmc@ee160000 --- salvator xs only

so, maybe we need to disable passthrough for them too

**Comment by oleksiimoisieiev** [comment]
+1
Acked-by: Oleksii Moisieiev <oleksii_moisieiev@epam.com>

I think if we will need extra SD slots for Dom0 It can be removed as a separate PR

**Comment by oleksiimoisieiev** [comment]
@Deedone  Please rebase this MR to develop

**Review by xakep-amatop — APPROVED** [review]
`Acked-by: Mykola Kvach <mykola_kvach@epam.com>`

File: src/domains/domd/domd_cfg_gen3.c [hunk 1]
```diff
@@ -26,7 +26,6 @@ static char* domd_dtdevs[] = {
     "/soc/usb@ee0a0000",
     "/soc/usb@ee0a0100",
     "/soc/mmc@ee100000",
-    "/soc/mmc@ee140000",
```
Comment (al1img) [inline src/domains/domd/domd_cfg_gen3.c:29]: Fix commit message: `boot`.

**Pull request merged**